hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py

```diff
@@ -171,12 +170,19 @@ def _theoretical_hours(self, employee, date):
         """
         if not employee.resource_id.calendar_id:
             return 0
-        tz = employee.resource_id.calendar_id.tz
-        return employee.with_context(
-            exclude_public_holidays=True, employee_id=employee.id
-        ).get_work_days_data(
```

Comment:
Why not change only get_work_days_data to _get_work_days_data

Reply:
thanks @ernestotejeda